### PR TITLE
fix query users when filter by email [prisma1->prisma2 style]

### PR DESCRIPTION
**Pull request by andriisoldatenko**
## Description

> Describe the purpose of this pull request.

## 🎟 Issue(s)

Resolves https://github.com/astronomer/issues/issues/1998

### CODE REVIEW

File: src/lib/users/index.js [hunk 1]
```diff
@@ -230,7 +230,7 @@ export function userQuery(args) {
   if (userUuid) return { id: userUuid };
 
   // If email, use it
-  if (email) return { emails_some: { address: email.toLowerCase() } };
+  if (email) return { emails: { some: { address: email.toLowerCase() } } };
```
Comment (andriisoldatenko) [inline src/lib/users/index.js:233]: prisma1/prisma2 mismatch.

File: src/lib/users/index.js [hunk 2]
```diff
@@ -230,7 +230,7 @@ export function userQuery(args) {
...
 
   // If username, use it
   if (username) return { username_contains: username.toLowerCase() };
```
Comment (schnie) [inline src/lib/users/index.js:236]: @andriisoldatenko does this need to be changed too?

Comment (andriisoldatenko) [reply src/lib/users/index.js:236]: oops yep!

**Comment by andriisoldatenko** [comment]
practically LGTM:
```
query users {
  users(user: {username: "test"}) {
		id
    username
  }
}

query users {
  users(user: {email: "test"}) {
		id
    username
  }
}

{
  "data": {
    "users": []
  }
}
```

**Comment by andriisoldatenko** [comment]
@adam2k @kushalmalani PTAL.

**Review by adam2k — APPROVED** [review]
👍 🎉 looks good!  Good catch.

**Review by kushalmalani — APPROVED** [review]

**Review by schnie — APPROVED** [review]

**Pull request merged**